notebooks/lesson02_exploratory-data-analysis.ipynb

```diff
@@ -0,0 +1,2971 @@
+{
```

Comment:
<p>we've never really introduced the index concept in pandas. maybe we should add a comment somewhere what it is and that it can also be other numbers than 0, 1, ..., n and even objects?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
<p>Done</p>